src/config.rs

```diff
@@ -149,19 +157,21 @@ impl StarshipConfig {
     fn config_from_file() -> Option<toml::Table> {
         let file_path = get_config_path()?;
 
-        let toml_content = match utils::read_file(file_path) {
+        let toml_content = match utils::read_file(&file_path) {
```

Comment:
Changed the move to a borrow so I can use the file_path for the error message later. Worked in my testing, but I'm not sure if it will break in other cases.